crates/matrix-sdk-base/src/rooms/normal.rs

```diff
@@ -115,13 +115,13 @@ impl Room {
     }
 
     /// Get the type of the room.
-    pub fn room_type(&self) -> RoomType {
-        self.inner.read().unwrap().room_type
+    pub fn room_type(&self) -> RoomState {
```

Comment:
Probably the method as well?

```suggestion
    pub fn room_state(&self) -> RoomState {
```